### PR TITLE
Add account registration gas amount calculation

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Add account registration gas amount 
- Add tests for:
  - Account registration gas amount
  - External RPC call gas amount

Related issue: https://github.com/ainblockchain/ain-blockchain/issues/316

### CODE REVIEW

File: integration/node.test.js [hunk 1]
```diff
@@ -2664,14 +2694,25 @@ describe('Blockchain Node', () => {
     })
 
     beforeEach(() => {
-      setUpForNativeFunctions();
+      setUpForFunctionTriggering();
+
+      // Setup mock for REST API calls.
+      const response = { 'success': true };
+      nock('https://events.ainetwork.ai')
```
Comment (liayoo) [inline integration/node.test.js:2701]: Cool! 😎

**Review by liayoo — APPROVED** [review]
LGTM, thanks! Btw, are you planning to apply the registration fee to service_accounts as well?

**Comment by platfowner** [comment]
@liayoo Actually, this change covers both individual and service account cases.
Just added test cases for service account registration gas amount.

**Comment by platfowner** [comment]
Thanks for the fast review!

**Pull request merged**